docs/src/openapi-v1.yaml

```diff
@@ -962,6 +962,114 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /pallets/{palletId}/errors:
+    get:
+      tags:
+      - pallets
+      summary: Get a list of errors for a pallet.
+      description: Returns a list of error item metadata for error items of the
+        specified palletId.
+      parameters:
+      - name: palletId
+        in: path
+        description: 'Name or index of the pallet to read error metadata for. Note: the pallet name must match what is
+          specified in the runtime metadata.'
+        required: true
+        schema:
+          type: string
+      - name: onlyIds
+        in: query
+        description: Only return the names (IDs) of the error items instead of all of each error
```

Comment:
Wording here is a little wonky.

Reply:
Ahh nvm, i get it now I missed the `items metadata` ending. But maybe instead of `of all of each error` we could do `of every error's`